shared/pkg/config/trivy.go

```diff
@@ -23,10 +23,14 @@ import (
 
 const AnalyzerTrivyTimeoutSecondsDefault = 300
 
-const AnalyzerTrivyTimeoutSeconds = "ANALYZER_TRIVY_TIMEOUT_SECONDS"
+const (
+	AnalyzerTrivyTimeoutSeconds = "ANALYZER_TRIVY_TIMEOUT_SECONDS"
+	AnalyzerTrivyCacheDir       = "ANALYZER_TRIVY_CACHE_DIRECTORY"
+)
 
 type AnalyzerTrivyConfig struct {
-	Timeout int `yaml:"timeout" mapstructure:"timeout"`
+	Timeout  int    `yaml:"timeout" mapstructure:"timeout"`
+	CacheDir string `yaml:"cache-dir" mapstructure:"cache-dir"`
```

Comment:
can it be `cache_dir` and not `cache-dir`?
I think we are using underscore in all other places